<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>##Author github user @Tanneguydv, 2021</author>
 <class>poccad</class>
 <widget class="QMainWindow" name="poccad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1330</width>
    <height>851</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>poccad</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ui_files/icons/poccad.png</normaloff>ui_files/icons/poccad.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPlainTextEdit" name="log">
       <property name="overwriteMode">
        <bool>false</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="activeview">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>ui_files/icons/Iso_on.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="isoview_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>ui_files/icons/Iso_off.png</normaloff>ui_files/icons/Iso_off.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="topview_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>ui_files/icons/Top_off.png</normaloff>ui_files/icons/Top_off.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="frontview_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>ui_files/icons/Front_off.png</normaloff>ui_files/icons/Front_off.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="leftview_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>ui_files/icons/Left_off.png</normaloff>ui_files/icons/Left_off.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bottomview_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>ui_files/icons/Bottom_off.png</normaloff>ui_files/icons/Bottom_off.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rightview_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>ui_files/icons/Right_off.png</normaloff>ui_files/icons/Right_off.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="rearview_button">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>ui_files/icons/Rear_off.png</normaloff>ui_files/icons/Rear_off.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="CodeEditor" name="OCCedit">
       <property name="tabChangesFocus">
        <bool>false</bool>
       </property>
       <property name="lineWrapMode">
        <enum>QPlainTextEdit::NoWrap</enum>
       </property>
       <property name="tabStopWidth">
        <number>20</number>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="OCClayout" stretch="0">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="render_button">
          <property name="text">
           <string>Render</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1330</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuShape">
    <property name="title">
     <string>Shape</string>
    </property>
    <addaction name="actionBox"/>
    <addaction name="actionSphere"/>
    <addaction name="actionRead_Step"/>
    <addaction name="actionPoint"/>
   </widget>
   <widget class="QMenu" name="menuBoolean">
    <property name="title">
     <string>Boolean</string>
    </property>
    <addaction name="actionCut"/>
   </widget>
   <widget class="QMenu" name="menuTransform">
    <property name="title">
     <string>Transform</string>
    </property>
    <addaction name="actionTranslate"/>
   </widget>
   <widget class="QMenu" name="menuConstruction">
    <property name="title">
     <string>Construction</string>
    </property>
    <addaction name="actionVector"/>
   </widget>
   <widget class="QMenu" name="menuExport">
    <property name="title">
     <string>Export</string>
    </property>
    <addaction name="actionStep"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actiondevmode"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuShape"/>
   <addaction name="menuBoolean"/>
   <addaction name="menuTransform"/>
   <addaction name="menuConstruction"/>
   <addaction name="menuExport"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBox">
   <property name="text">
    <string>Box</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSphere">
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionRead_Step">
   <property name="text">
    <string>Read Step</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPoint">
   <property name="text">
    <string>Point</string>
   </property>
  </action>
  <action name="actionVector">
   <property name="text">
    <string>Vector</string>
   </property>
  </action>
  <action name="actionTranslate">
   <property name="text">
    <string>Translate</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="text">
    <string>Step</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>info</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actiondevmode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>devmode</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CodeEditor</class>
   <extends>QPlainTextEdit</extends>
   <header>lib.codeeditor.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
